.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
because I spent hours figuring out that the prettier config path was set to "./" in my settings and that completely doesn't work and isn't obvious. I don't know if this is a VSC update problem or just me.